src/core/reducers/autocomplete.js

```diff
@@ -0,0 +1,69 @@
+export const AUTOCOMPLETE_LOADED = 'core/autocomplete/AUTOCOMPLETE_LOADED';
+export const AUTOCOMPLETE_STARTED = 'core/autocomplete/AUTOCOMPLETE_STARTED';
+export const AUTOCOMPLETE_CANCELLED = 'core/autocomplete/AUTOCOMPLETE_CANCELLED';
+
+const initialState = {
+  loading: false,
+  suggestions: [],
```

Comment:
I am ok with that, it should not require too much extra code.

I did not do that because autocompletion looks like a "secondary" feature to me: if no result is shown to the user for some reasons, that's fine because she is typing in the search input anyway. Nothing is really broken because user is looking for something _via_ the search input and relying on it for quick/direct access to add-ons is still not natural/common (IMO).